### DISCUSSION TITLE
Distributed social network

### DISCUSSION THREAD

**Discussion by connorsml** [ryanhowdy/fcms#664] [Ideas]
Hi! I stumbled across this project while looking to find a way to host my family tree in a way I could share it without using Ancestry or Geni. It looks great.
I know this is something probably way beyond the scope of a project like this but has any thought been given to a distributed version of fcms.
We have an immediate family, in-laws and more distant relations. Everyones circles overlap. If I had a node and someone on my in-laws side had a node it would be cool if parts of the content could be shared between them.

**Comment by ryanhowdy** [comment]
That is a great idea and something I need to think about.

**Reply by connorsml** [reply]
Perhaps with Activity Pub which powers Mastodon, Pixelfed and other distributed social networks there is scope to do something with this?
